src/data/faq.json

```diff
@@ -2063,12 +2073,40 @@
                     "description": "See DCC"
                 }
             ],
+            "P": [
+            {
+                "term": "Proof of vaccination",
+                "description": "The proof of vaccination documents a vaccination procedure with a vaccine recommended by the Paul-Ehrlich-Institut. A vaccination proof must contain at least the following attributes: <ul><li>the personal data of the vaccinated person (at least surname, first name and date of birth)</li><li>date of vaccination, number of vaccinations,</li><li>designation of the vaccine,</li><li>name of the disease against which the vaccination was carried out, and</li><li>characteristics that indicate the person or institution responsible for carrying out the vaccination or issuing the proof of vaccination, for example, an official symbol or the name of the issuer. </li></ul> <br/><br/> For entry, according to § 2 number 10 Coronavirus-Einreiseverordnung, the vaccination certificate must still meet additional requirements (regarding languages and vaccination schema). <br/><br/> The proof of vaccination can be documented in different ways, one form is the digital => vaccination certificate."
+            }
+            ],
+            "R": [
+                {
+                    "term": "Recovery Certificate",
+                    "description": "The certificate of recovery is a => digital certificate issued to a person who has demonstrably recovered from COVID-19 and proved immunity by a PCR test."

```

Comment:
issued to --> issued for

Reply:
Done via https://github.com/corona-warn-app/cwa-website/commit/529b7e25f68173e5ffeabe6c3968492bb6d8f967.